pkg/infra/iac2/templates_compiler.go

```diff
@@ -278,20 +259,24 @@ func (tc TemplatesCompiler) resolveDependencies(resource core.Resource) string {
 	numDeps := len(upstreamResources)
 	for i := 0; i < numDeps; i++ {
 		res := upstreamResources[i]
-		buf.WriteString(tc.getVarName(res))
-		if i < (numDeps - 1) {
-			buf.WriteRune(',')
+		switch res.(type) {
+		case *resources.Region, *resources.AvailabilityZones, *resources.AccountId:
```

Comment:
(non-blocking) I wonder if there's a better way to tell which resources to ignore rather than hardcoding.

Reply:
yeah im not sure, but its iac specific so i wouldnt want to include it on the actual resource. i also dont think it will be more than just these 3 for a while unless we end up having more special cased things